utils/rpms/daos.spec

```diff
@@ -364,6 +368,7 @@ getent passwd daos_server >/dev/null || useradd -M daos_server
 %{_prefix}/lib/cart/TESTING
 %{_bindir}/hello_drpc
 %{_bindir}/*_test*
+%exclude %{_bindir}/self_test
```

Comment:
why are we excluding self_test?

Reply:
Because it's included in the client:
https://github.com/daos-stack/daos/blob/591a64110a355e3bafbfb29f5fe7be7e8ab373ea/utils/rpms/daos.spec#L307